packages/eslint-plugin/package.json

```diff
@@ -46,6 +46,7 @@
     "@typescript-eslint/scope-manager": "4.12.0",
     "debug": "^4.1.1",
     "functional-red-black-tree": "^1.0.1",
+    "lodash": "^4.17.15",
```

Comment:
`@typescript-eslint/eslint-plugin-tslint` and `@typescript-eslint/typescript-estree` already depend on it, so shouldn't make a difference